src/index.js

```diff
@@ -276,6 +287,11 @@ export function hamburgerTransform() {
     document.getElementById('hamburger_bottom').classList.remove(
       'transform_bottom'
     );
+    let placeholder = document.getElementById('placeholder');
+    let visibleProjectName = placeholder.innerHTML.split(' ').join('');
+    // Once the student has hidden the test window, this localStorage variable
+    // keeps it hidden until manually toggled.
+    localStorage.setItem('fCC_' + visibleProjectName + '_hide', true);
```

Comment:
I don't see where this gets set to false. What am I missing?

Also see my comment above about making the building of this string DRY.

Reply:
It doesn't get set to false. Once hidden, the window stays hidden until toggled for each page load.